components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/utils/SCIMConfigProcessorTest.java

```diff
@@ -0,0 +1,121 @@
+    }
+
+    @BeforeMethod public void setUp() throws Exception {
+        scimConfigProcessor = new SCIMConfigProcessor();
+
+        scimConfigProcessor.properties.put("Name", "SCIM2");
+        scimConfigProcessor.properties.put("Age", "24");
+
+        authenticationSchema = new AuthenticationSchema();
+        authenticationSchema.setName("SCIM");
+        authenticationSchema.setDescription("SCIM2");
+        authenticationSchema.setSpecUri("https://localhost:9443/scim2");
+        authenticationSchema.setDocumentationUri("https://localhost:9443/scim2/docs");
+        authenticationSchema.setType("Authentication");
+        authenticationSchema.setPrimary("true");
+        scimConfigProcessor.authenticationSchemas = new ArrayList<>();
+        scimConfigProcessor.authenticationSchemas.add(authenticationSchema);
+    }
+
+    @AfterMethod public void tearDown() throws Exception {
+    }
+
+    @Test public void testGetProperties() throws Exception {
+        Map<String, String> expected = new HashMap<String, String>();
+        expected.put("Name", "SCIM2");
+        expected.put("Age", "24");
+
+        Map<String, String> map = scimConfigProcessor.getProperties();
+
+        assertEquals(map.get("Name"), expected.get("Name"));
+        assertEquals(map.get("Age"), expected.get("Age"));
+    }
+
+    @Test(dataProvider = "facebook-authenticator-all") public void testGetProperty(String property,
+            String expectedResult) throws Exception {
+        assertEquals(scimConfigProcessor.getProperty(property), expectedResult);
+    }
+
+    @Test public void testGetAuthenticationSchemas() throws Exception {
+        List<AuthenticationSchema> authenticationSchemaList = scimConfigProcessor.getAuthenticationSchemas();
+        for (AuthenticationSchema authenticationSchema1 : authenticationSchemaList) {
+            assertEquals(authenticationSchema1.getName(), authenticationSchema.getName());
+            assertEquals(authenticationSchema1.getDescription(), authenticationSchema.getDescription());
+            assertEquals(authenticationSchema1.getSpecUri(), authenticationSchema.getSpecUri());
+            assertEquals(authenticationSchema1.getDocumentationUri(), authenticationSchema.getDocumentationUri());
+            assertEquals(authenticationSchema1.getType(), authenticationSchema.getType());
+            assertEquals(authenticationSchema1.getPrimary(), authenticationSchema.getPrimary());
+        }
+    }
+
+    @Test(dataProvider = "filePathProvider", expectedExceptions = CharonException.class) public void testBuildConfigFromFile(
+            String filePath) throws Exception {
+        scimConfigProcessor.buildConfigFromFile(filePath);
+    }
+
+    @Test public void testBuildConfigFromFileHappy() throws Exception {
+        String filePath = Paths
+                .get(System.getProperty("user.dir"), "src", "test", "resources", "charon-config-test.xml").toString();
```

Comment:
Unnecessary new line.

Reply:
When I format the code from the wso2 code styles, the code becomes like that.